frontend/src/App.svelte

```diff
@@ -68,6 +70,49 @@
     }
   }
 
+
+                                                               // non-interactive mode variables
+  let show_message:boolean = false; 
```

Comment:
consider formatting, var:T needs to become var: T
and for the lines below this, curly braces should not touch other things.

Reply:
It is still not satisfactory but as we discussed about it, I decided to forego formatting issues until we figured out auto formatting. Though I hope you understand that even with no formatter you should still write consistent code.